models/user.py

```diff
@@ -44,7 +53,11 @@ def __init__(self, first_name, last_name, username, password):
             :param password: strong password (strength can be checked with is_strong)
 		    :type password: str
         '''
-        pass
+        self.first_name = first_name
+        self.last_name = last_name
+        self.username = username
+        self.__password = password
+
```

Comment:
Good but your __init__ function is missing sth to be able to inherit from the Model class